src/Statistics/EmployeeParticipationShare.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics;
+
+use VSV\GVQ_API\Company\ValueObjects\PositiveNumber;
+
+class EmployeeParticipationShare
+{
+    /**
+     * @var PositiveNumber
+     */
+    private $totalEmployees;
+
+    /**
+     * @var int
+     */
+    private $participationCount;
+
+    /**
+     * @param PositiveNumber $totalEmployees
+     * @param int $participationCount
+     */
+    public function __construct(int $participationCount, PositiveNumber $totalEmployees)
+    {
+        $this->totalEmployees = $totalEmployees;
```

Comment:
Je zou hier een guard method kunnen opzetten die gewoon een InvalidArgumentException throwed als `participationCount` kleiner is dan 0.